src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -507,6 +513,234 @@ def _test_flexible_server_georestore_mgmt(self, database_engine, resource_group)
         self.cmd('{} flexible-server delete -g {} -n {} --yes'.format(
                  database_engine, resource_group, target_server_public_access_2), checks=NoneCheck())
 
+    def _test_flexible_server_byok_mgmt(self, database_engine, resource_group):
+        vault_name = self.create_random_name('rdbmsvault', 24)
+        key_name = self.create_random_name('rdbmskey', 32)
+        identity_name = self.create_random_name('identity', 32)
+        backup_vault_name = self.create_random_name('rdbmsvault', 24)
+        backup_key_name = self.create_random_name('rdbmskey', 32)
+        backup_identity_name = self.create_random_name('identity', 32)
+        server_name = self.create_random_name(SERVER_NAME_PREFIX, SERVER_NAME_MAX_LENGTH)
+        replica_1_name = self.create_random_name(SERVER_NAME_PREFIX, SERVER_NAME_MAX_LENGTH)
+        replica_2_name = self.create_random_name(SERVER_NAME_PREFIX, SERVER_NAME_MAX_LENGTH)
+        backup_name = self.create_random_name(SERVER_NAME_PREFIX, SERVER_NAME_MAX_LENGTH)
+        tier = 'GeneralPurpose'
+        sku_name = 'Standard_D2ds_v4'
+        location = 'northeurope'
+        backup_location = 'westeurope'
+        backup_location_full_name = 'West Europe'
+
+        self.cmd(
+            'keyvault create -g {} -n {} --location {} --enable-soft-delete true --enable-purge-protection true'
```

Comment:
Can you use `@KeyvaultPreparer(name_prefix='rdbmsvault', key='vault_name', location='northeurope', additional_params='--enabled-for-deployment true --enabled-for-template-deployment true')` instead?